### PR TITLE
 new release 0.4.4

### PR DESCRIPTION
**Pull request by medo-freiheit**

### CODE REVIEW

File: charts/kuberpult/values.yaml [hunk 1]
```diff
@@ -61,6 +61,9 @@ argocd:
 datadogTracing:
   enabled: false
   debugging: false
-  metrics: false
+
+dogstatsdMetrics:
+  enabled: false
+  address: "127.0.0.1:8125"
```
Comment (fdcds) [inline charts/kuberpult/values.yaml:67]: I would add a link to the documentation here, to explain why port 8125. Explaining that here in text would be nice for someone like me, but I would consider it not critical.

```suggestion
  # dogstatsD listens on port udp:8125 by default.
  # https://docs.datadoghq.com/developers/dogstatsd/?tab=hostagent#agent
  address: "127.0.0.1:8125"
```

File: charts/kuberpult/values.yaml [hunk 2]
```diff
@@ -61,6 +61,9 @@ argocd:
 datadogTracing:
   enabled: false
   debugging: false
-  metrics: false
```
Comment (fdcds) [inline charts/kuberpult/values.yaml:64]: Q: This was unused before?

**Review by fdcds — COMMENTED** [review]
LGTM, but I am not familiar with the code yet, so I feel not comfortable to actually approve this.

I have one question and a small suggestion.

File: charts/kuberpult/values.yaml (hunk 2 above)
Comment (medo-freiheit) [reply charts/kuberpult/values.yaml:64]: it's related to my new story, I added it but didn’t adapt the helm chart with it, I did that in this PR so I just reorganized it

File: charts/kuberpult/values.yaml (hunk 1 above)
Comment (medo-freiheit) [reply charts/kuberpult/values.yaml:67]: Fair enough, I will add this

**Pull request merged**